File: uebung_1/tast_1/prettyOut.cpp
#include <iostream>
using namespace std;
void ausgabe(char c);
void ausgabe(char *c);

int main()
{
  ausgabe('a');
  ausgabe("hallo welt");
  return 0;
}

void ausgabe(char c)
{
  cout << "***\n*" << c << "*\n***\n";
}

void ausgabe(char *c)
{
  int length = strlen(c);
  string border = "**";
  for (int i = 0; i < length; i++)
  {
    border += "*";
  }

  cout << border << "\n*" << c << "*\n"
       << border << "\n";
}

File: uebung_1/task_1/PrettyOut.java
package uebung_1.task_1;

public class PrettyOut {
  public static void main(String[] args) {
    ausgabe('d');
    ausgabe("hallo welt");

  }

  static void ausgabe(char c) {
    System.out.println("***\n*" + c + "*\n***");
  }

  static void ausgabe(String text) {
    String border = "**";
    for (int i = 0; i < text.length(); i++) {
      border += "*";
    }
    System.out.println(border + "\n*" + text + "*\n" + border);
  }
}